### DISCUSSION TITLE
Help with importing autogluon.eda v0.7.0

### DISCUSSION THREAD

**Discussion by bustosxaus** [autogluon/autogluon#3086] [Q&A] [answered]
Hi, sorry if this is not the right forum for this question, I installed autogluon v0.7.0 and tried to import autogluon.eda, but got the following error:

`ModuleNotFoundError: No module named 'autogluon.eda'`

Can anyone please help me or guide me in the right direction?

Many thanks!

**Comment by bustosxaus** [comment, marked as answer]
Sorry, I pip installed autogluon.eda directly, is it going to be included in autogluon's next release? Thank you

**Reply by Innixma** [reply]
We will consider adding it as a default install in an upcoming release. For now, it is experimental, so we have not added it by default. This is mostly to avoid developer efforts on EDA impacting the stability of AutoGluon as a whole.

**Comment by bustosxaus** [comment]
Thank you very much for the quick answer and explanation, Nick, looking forward to future releases!
